lib/semantics/check-do.h

```diff
@@ -28,7 +28,7 @@ namespace Fortran::semantics {
 
 // To specify CYCLE and EXIT statements in semantic checking that's common to
 // both of them.
```

Comment:
Comment is outdated (should be broader than `CYCLE` and `EXIT` now)

Reply:
Thanks for catching this.  I'll update the comment.